ec/README.md

```diff
@@ -14,7 +14,6 @@ The available elliptic curve traits are:
 * [`AffineCurve`](https://github.com/arkworks-rs/algebra/blob/master/ec/src/lib.rs#L223) - Interface for elliptic curve points in the 'canonical form' for serialization.
 * [`ProjectiveCurve`](https://github.com/arkworks-rs/algebra/blob/master/ec/src/lib.rs#L121) - Interface for elliptic curve points in a representation that is more efficient for most computation.
 * [`PairingEngine`](https://github.com/arkworks-rs/algebra/blob/master/ec/src/lib.rs#L44) - Pairing friendly elliptic curves (Contains the pairing function, and acts as a wrapper type on G1, G2, GT, and the relevant fields).
-* [`CycleEngine`](https://github.com/arkworks-rs/algebra/blob/master/ec/src/lib.rs#L318) - Wrapper type for a cycle of pairing friendly elliptic curves.
```

Comment:
Didn't we replace this with `CurveCycle` and `PairingFriendlyCycle`? Maybe instead of deleting it we should replace `CycleEngine` with this?

Reply:
Good points. Let me update the README

Reply:
Any feedback? I added two lines and just also updated the line numbers.
```
* [`CurveCycle`](https://github.com/arkworks-rs/algebra/blob/master/ec/src/lib.rs#L319) - Wrapper type for a cycle of elliptic curves.
* [`PairingFriendlyCycle`](https://github.com/arkworks-rs/algebra/blob/master/ec/src/lib.rs#L331) - Wrapper type for a cycle of pairing friendly elliptic curves. 
```

Reply:
Maybe say "Trait representing a cycle of..."?

Reply:
Done!